src/homeWork16/FileNavigator.java

```diff
@@ -0,0 +1,62 @@
+package homeWork16;
+
+import java.util.*;
+
+public class FileNavigator {
+
+    private Map<String, List<FileData>> fileMap = new HashMap<>();
+
+    public void add(String path, FileData fileData) {
```

Comment:
Check whether the arguments are null or not

Reply:
+++